project/profiles/capacity4more/themes/c4m/kapablo/templates/page.tpl.php

```diff
@@ -93,103 +93,109 @@
     </section>
   <?php endif; ?>
 
-  <!--  TOP MENU -->
```

Comment:
@anvmn 
Can't we just use a different template instead of changing `page.tpl.php` ?